### ISSUE TITLE
Build instructions for Linux gone?

### ISSUE THREAD

**Issue by tharbad08** [tracktwo/xcomsave#23]
Just in case:

1. cd xcomsave
2. cmake . 
3. make

Then just use xcom2json and json2xcom. They'll be in the root folder

Checked and working on Arch.

**Comment by tracktwo** [comment]
Yeah I didn't add specific build instructions for any platform, but it's a good idea to add them. My preferred way is to build from a build directory, so the root doesn't get polluted with more files, but your way definitely works too.

```
cd xcomsave
mkdir build
cd build
cmake ..
make
```
